src/domain/file/services/file-service.ts

```diff
@@ -70,17 +74,28 @@ export class FileService {
 
         try {
             const params = await this.s3.uploadPart(partParams).promise();
-            await statusFunction(userId, file.filename, file.id, Math.floor((bytesRead / file.size) * 100));
+            await pubsub.publish('UPLOAD_STATUS', {
+                fileUploadProgress: {
+                    userId,
+                    filename: file.filename,
+                    fileId: file.id,
+                    percentage: Math.floor((bytesRead / file.size) * 100),
+                    type: FileType.MANUSCRIPT_SOURCE,
```

Comment:
Shouldn't this be the type passed in?

Reply:
Yes, good point. Didn't refactor it as nicely as I had thought

Reply:
Will push up as part of the fix for failing tests